ui/packages/atlasmap-core/src/contracts/AtlasmapLanguage.tsx

```diff
@@ -0,0 +1,306 @@
+/*
+    Copyright (C) 2017 Red Hat, Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+import { languages } from 'monaco-editor';
+
+export const atlasmapLanguageID = 'atlasmapLanguage';
+export const atlasmapWordPattern =
+  /[a-zA-Z0-9_\-*)|([^`~!@#%^&*()\-=+[{\]}\\|;:'",.<>/?\s]+/g;
+
+export const atlasmapLanguageConfig = {
```

Comment:
So this will be removed or drastically reduced once #4138 is addressed, correct? maybe put a comment right here about that

Reply:
done